pimd/pim_cmd.c

```diff
@@ -3415,15 +3415,25 @@ static void igmp_show_groups(struct pim_instance *pim, struct vty *vty, bool uj)
 	time_t now;
 	json_object *json = NULL;
 	json_object *json_iface = NULL;
-	json_object *json_row = NULL;
+	json_object *json_group = NULL;
+	json_object *json_groups = NULL;
 
 	now = pim_time_monotonic_sec();
 
-	if (uj)
+	if (uj) {
 		json = json_object_new_object();
-	else
+		json_object_int_add(json, "Total Groups",
+				    pim->igmp_group_count);
+		json_object_int_add(json, "Watermark limit",
```

Comment:
keys are to be camelCased without spaces

Reply:
changed to 'totalGroups' and 'watermarkLimit'